### PR TITLE
Block Writer example provides `/health` endpoint

### PR DESCRIPTION
**Pull request by mrtracy**
The Block Writer example now provides a `/health` endpoint over http, allowing
it to be monitored by cockroach's status monitor tool. This will support our
ability to run automated tests in a well-coordinated fashion.

[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/examples-go/23)

### CODE REVIEW

File: block_writer/main.go [hunk 1]
```diff
@@ -98,6 +104,22 @@ func (bw blockWriter) randomBlock() []byte {
 	return blockData
 }
 
+// handleHealth handles incoming requests to the '/health' endpoint, allowing
+// this service to be monitored over http.
+func handleHealth(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprint(w, "Service healthy.")
```
Comment (mberhault) [inline block_writer/main.go]: just plain `OK` is fairly standard as the response contents.

**Comment by mberhault** [comment]
LGTM

**Comment by tamird** [comment]
LGTM. Any reason this hasn't been merged yet?

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[block_writer/main.go, line 117 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/examples-go/23#-KC1EQxc1kEDjcjtG9PP:-KC1EQxc1kEDjcjtG9PQ:-982455560) ([raw file](https://github.com/cockroachdb/examples-go/blob/1d53033e927beee580ab94a2386cec6bced189a0/block_writer/main.go#L117)):*
one line it

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/examples-go/23#-:-KC1ETaMNDQ1zmOMpX4v:1318229201)*

**Comment by mrtracy** [comment]
I think right after I sent it, it was no longer needed for anything. This was when I was working on setting up automated cluster tests, but i was quickly back to other types of work.  I'm just going to close it for now.

**Comment by tamird** [comment]
Sounds good to me.

**Pull request closed without merging**